core/src/main/protobuf/sonarlint.proto

```diff
@@ -68,9 +66,24 @@ message Rules {
     bool is_template = 8;
     string template_key = 9;
   }
-  
+}
+
+message QProfiles {
+  map<string,QProfile> qprofilesByKey = 1;
+  map<string, string> defaultQProfilesByLanguage = 2;
+
   message QProfile {
     string key = 1;
+    string name = 2;
+    string language = 3;
+    string languageName = 4;
+    bool isInherited = 5;
```

Comment:
We only need key and rule count. The rest was for future use, but since we'll be versioning the storage, doesn't make sense anymore.